discord_bots/automation_bot/automation_bot_instance.py

```diff
@@ -2354,6 +2402,10 @@ async def action_weekly_update(self, ctx: discord.ext.commands.Context, cmd, mes
             await self.display(ctx, "Running item replacements for players...")
             await self.run(ctx, os.path.join(_top_level, "utility_code/weekly_update_player_data.py") + f" --world {self._server_dir}/server_config/redis_data_initial --datapacks {self._server_dir}/server_config/data/datapacks -j {config.CPU_COUNT}")
 
+            # TODO REMOVE THIS
+            await self.display(ctx, "Stopping for manual player data fixes")
+            return
+
```

Comment:
This definitely shouldn't be checked in - probably you grabbed a snapshot of the repo at some time when this was an uncommitted local change